src/status_im/chat/views/input/input.cljs

```diff
@@ -16,13 +16,13 @@
             [status-im.chat.views.input.validation-messages :as validation-messages]
             [status-im.ui.components.animation :as anim]
             [status-im.ui.components.react :refer [view
-                                                animated-view
-                                                text
-                                                icon
-                                                scroll-view
-                                                text-input
-                                                touchable-highlight
-                                                dismiss-keyboard!]]
+                                                   animated-view
```

Comment:
why do we have a lot of this? who did this?

Reply:
just asking, maybe he should adjust his alignments rules :)

Reply:
Cursive I guess.

Reply:
we should get rid of refers and use :as anyway

Reply:
Right